optimism/src/keccak/column.rs

```diff
@@ -76,29 +80,29 @@ pub enum KeccakColumn {
     Output(usize),          // Next[0..100) either IotaStateG or SpongeXorState
 }
 
-/// The columns used by the Keccak circuit.
-/// The Keccak circuit is split into two parts: Sponge and Round.
+/// The witness columns used by the Keccak circuit.
+/// The Keccak circuit is split into two main modes: Sponge and Round.
 /// The columns are shared between the Sponge and Round steps.
-/// The step index and hash index are shared between the Sponge and Round step
-/// type.
+/// The hash and step indices are shared between both modes.
 #[derive(Clone, Debug, Eq, PartialEq)]
-pub struct KeccakColumns<T> {
-    pub hash_index: T,
-    pub step_index: T,
+pub struct KeccakWitness<T> {
+    pub hash_index: T,                        // Which hash this is inside the circuit
```

Comment:
Follow-up: I would explain somewhere why the index is important. We did not have it before talking about the syscall communication channel. It can be in the code or in the RFC (and we point to the RFC here).

Reply:
I will address that in the following chunk of docs